support.html

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: Get Support
+nav: floating
+---
+
+<div class="section narrow lightest-text" style="margin-bottom: -4rem;">
+	<div class="container">
+		<div class="row">
+			<div class="column two-thirds left">
+				<h1>Get Support</h1>
+				<p>Need help? No problem! Reach out to our community on <a href="//forum.defold.com">forum</a> and <a href="/discord">discord</a>. Additionally, at Defold we offer expert guidance, resources, and assistance to help ensure the long-term success of your game.</p>
```

Comment:
Should we make it clear that we're part of the community as well? Something like:

> Need help? No problem! The developers frequently join our helpful community on the forum and discord. Additionally, at Defold we offer expert guidance, resources, and assistance to help ensure the long-term success of your game.